src/App.js

```diff
@@ -48,6 +69,8 @@ class App extends Component {
 							}
 						}}
 					/>
+					<Route exact path='/MovieShowPage/:movieId'
+					render={this.findMovieShowInfo}/>
```

Comment:
Great job setting up the router to pass down the info from the movie through the helper method!